docs/src/docs/welcome/faq.mdx

```diff
@@ -45,10 +45,11 @@ If there are `typecheck` errors, golangci-lint will not be able to produce other
 How to troubleshoot:
 
 - [ ] Ensure the version of golangci-lint is built with a compatible version of Go.
-- [ ] Ensure dependencies are up-to-date with `go mod tidy`.
+- [ ] Ensure dependencies are up to date with `go mod tidy`.
 - [ ] Ensure building works with `go run ./...`/`go build ./...` - whole package.
 - [ ] Ensure you are not running an analysis on code that depends on files/packages outside the scope of the analyzed elements.
 - [ ] If using CGO, ensure all require system libraries are installed.
+- [ ] If you are using private repositories/dependencies, ensure the Go proxy and sumDB are right and/or your git configuration.
```

Comment:
I suggest adding links to this since I think not everyone is aware of what this implies.

```suggestion
- [ ] If you are using private repositories/dependencies, ensure the [Go proxy](https://go.dev/ref/mod#module-proxy) (`GOPROXY`) and [checksum database](https://go.dev/ref/mod#checksum-database) (`GOSUMDB`) are right and/or your git configuration.
```